package/zmq_router/src/zmq_router_sbp.c

```diff
@@ -214,6 +222,27 @@ static port_t ports_sbp[] = {
     },
     .pub_socket = NULL,
     .sub_socket = NULL,
+  },
+  [SBP_PORT_SKYLARK] = {
+    .config = {
+      .pub_addr = "@tcp://127.0.0.1:43070",
+      .sub_addr = "@tcp://127.0.0.1:43071",
+      .sub_forwarding_rules = (const forwarding_rule_t *[]) {
+        &(forwarding_rule_t){
+          .dst_port = &ports_sbp[SBP_PORT_FIRMWARE],
+          .filters = (const filter_t *[]) {
+            &FILTER_ACCEPT(0x55, 0x44, 0x00), /* Subscribe: SBP MSG_BASE_POS_LLH */
+            &FILTER_ACCEPT(0x55, 0x48, 0x00), /* Subscribe: SBP MSG_BASE_POS_ECEF */
+            &FILTER_ACCEPT(0x55, 0x4A, 0x00), /* Subscribe: SBP MSG_OBS */
```

Comment:
I think we should plan on on decimating observations to 1hz or similar no matter how the rover is configured.  I worry if a user puts raw data rate on at 20hz that it will be bad.